### PR TITLE
Feat/fti 4428 privileged dp connection

### PR DESCRIPTION
**Pull request by liverpool8056**
## Summary

The purpose of this pr is to back port #9432 to 2.8.x

### CODE REVIEW

File: kong/clustering/data_plane.lua [hunk 1]
```diff
@@ -18,6 +18,7 @@ local tostring = tostring
 local ngx = ngx
 local ngx_log = ngx.log
 local ngx_sleep = ngx.sleep
+local ngx_process = ngx.process
```
Comment (chronolaw) [inline kong/clustering/data_plane.lua:21]: should be `local process_type = ngx.process.type`

**Comment by chronolaw** [comment]
Need a change log entry.

File: kong/clustering/data_plane.lua (hunk 1 above)
Comment (liverpool8056) [reply kong/clustering/data_plane.lua]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -80,6 +87,19 @@
...
+#### Core
+
+- Supports boths 2.x route path and 3.0 route paths.
+  [#9376](https://github.com/Kong/kong/pull/9376) 
+- fetch ecs environment variable failure caused by the phase
+  [#9460](https://github.com/Kong/kong/pull/9460)
+- Start only needed upstream update timers 
+  [#9500](https://github.com/Kong/kong/pull/9500)
+- Dynamic router lru cache size based on number of routes
+  [#9475](https://github.com/Kong/kong/pull/9475)
```
Comment (mayocream) [inline CHANGELOG.md:101]: Git conflict?

File: kong-2.8.2-0.rockspec [hunk 3]
```diff
@@ -1,10 +1,10 @@
 package = "kong"
-version = "2.8.1-0"
+version = "2.8.2-0"
```
Comment (mayocream) [inline kong-2.8.2-0.rockspec:2]: We should bump the version in another PR.

File: Jenkinsfile [hunk 4]
```diff
@@ -11,9 +11,9 @@ pipeline {
         DOCKER_PASSWORD = "${env.DOCKER_CREDENTIALS_PSW}"
         KONG_PACKAGE_NAME = "kong"
         DOCKER_CLI_EXPERIMENTAL = "enabled"
-        PULP_HOST_PROD = "https://api.pulp.konnect-prod.konghq.com"
+        PULP_HOST_PROD = "https://api.download.konghq.com"
```
Comment (mayocream) [inline Jenkinsfile:14]: Why is this needed for this PR?

**Comment by mayocream** [comment]
As discussed with @liverpool8056 offline, we should postpone this PR.

**Comment by hbagdi** [comment]
Closing in that case. Please re-open if needed.
